src/nick-bot-commands.js

```diff
@@ -0,0 +1,47 @@
+import {findCommandFiles, getRestInstance} from "./shared.js";
+import {Routes} from "discord-api-types/v9";
+
+/**
+ * Purge all guild specific commands
+ * @returns {Promise<void>}
```

Comment:
Did you leave the `@param` out of here on purpose? You added them on the `deployCommands`